website/prebuild.py

```diff
@@ -146,12 +152,6 @@ def main() -> None:
     # Ensure this runs from the repo root.
     os.chdir(Path(__file__).parents[1])
 
-    # bazel-execroot interferes with jekyll because it's a broken symlink.
-    Path("bazel-execroot").unlink()
-    # This is a symlink to website/favicon.png, which is moved below.
-    # TODO: Consider moving the icon to a location which won't break.
-    Path("utils/vscode/images/icon.png").unlink()
```

Comment:
Note,. I think these were required to build, but maybe I'm mistaken?

Reply:
Yeah, I tested it and the exclude works to prevent problems with dangling symlinks.